README.md

```diff
@@ -1,8 +1,60 @@
-# Welcome to BEAR-Necessities-Market
+<p align="center"><img width=18% src="https://github.com/hawzie197/BEAR-Necessities-Market/blob/master/media/logo.png"></p>
+<p align="center"><img width=65% src="https://github.com/hawzie197/BEAR-Necessities-Market/blob/master/media/logo-text.png"></p>
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
 [![GitHub license](https://img.shields.io/badge/license-GPL%20V3-blue.svg)](https://github.com/LibreFoodPantry/BEAR-Necessities-Market/blob/master/LICENSE.md)
 [![Build Status](https://travis-ci.com/LibreFoodPantry/BEAR-Necessities-Market.svg?branch=master)](https://travis-ci.com/LibreFoodPantry/BEAR-Necessities-Market)
+[![Python 3.6](https://img.shields.io/badge/python-3.6-blue.svg)](https://www.python.org/downloads/release/python-360/)
+[![npm version](https://img.shields.io/npm/v/react.svg?style=flat)](https://www.npmjs.com/package/react)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://reactjs.org/docs/how-to-contribute.html#your-first-pull-request)
+[![Known Vulnerabilities](https://snyk.io/test/github/dwyl/hapi-auth-jwt2/badge.svg?targetFile=package.json)](https://snyk.io/test/github/dwyl/hapi-auth-jwt2?targetFile=package.json)
+![Dependencies](https://img.shields.io/badge/dependencies-up%20to%20date-brightgreen.svg)
+
+
+## Overview
+
+Some overview description will go here
+
+
+## Table of Contents
+
+- [Quick Setup](#quick-setup)
+- [Tech Stack](#tech-stack)
+- [Code Structure](#code-structure)
+- [Documentation][documentation]
+- [Releases][releases]
+- [Discussion](#discussion)
+- [License](#license)
+
+
+### Quick Setup
+
+> Or take a look at our detailed [getting started][getting-started] instructions.
+
+First, get a recent version of [Node.js][node] and [Python 3.4+][python]
+
+```bash
+
+curl -o- -L https://yarnpkg.com/install.sh | bash -s
+git clone https://github.com/LibreFoodPantry/BEAR-Necessities-Market.git
+cd BEAR-Necessities-Market/fronend
+
+yarn
+yarn run build
+cd ..
 
-## [React v16.8.1](https://facebook.github.io/react/) Frontend
+pip install -r requirements.txt
+export FLASK_APP=manage.py
+flask db upgrade
```

Comment:
I'm getting errors when I run `flask db upgrade`.

```bash
(BEAR-Necessities-Market) bash-5.0$ export FLASK_APP=manage.py
(BEAR-Necessities-Market) bash-5.0$ flask db upgrade
Map([<Rule '/auth/token/refresh/' (HEAD, GET, OPTIONS) -> auth_token_refresh>,
 <Rule '/auth/register/' (OPTIONS, POST) -> auth_registration_view>,
 <Rule '/auth/login/' (OPTIONS, POST) -> auth_login_view>,
 <Rule '/swagger.json' (HEAD, GET, OPTIONS) -> specs>,
 <Rule '/users/' (HEAD, GET, OPTIONS) -> users_users_list>,
 <Rule '/' (HEAD, GET, OPTIONS) -> default.index>,
 <Rule '/' (HEAD, GET, OPTIONS) -> doc>,
 <Rule '/' (HEAD, GET, OPTIONS) -> root>,
 <Rule '/swaggerui/<filename>' (HEAD, GET, OPTIONS) -> restplus_doc.static>,
 <Rule '/users/<pk>' (HEAD, GET, OPTIONS, DELETE) -> users_users_detail>,
 <Rule '/build/<filename>' (HEAD, GET, OPTIONS) -> default.static>,
 <Rule '/<filename>' (HEAD, GET, OPTIONS) -> static>])
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
INFO  [alembic.runtime.migration] Running upgrade a7852c8b3c34 -> 6ca021205583, empty message
ERROR [alembic.env] (sqlite3.OperationalError) table users already exists [SQL: '\nCREATE TABLE users (\n\tid INTEGER NOT NULL, \n\temail VARCHAR, \n\tpassword BLOB, \n\tauthenticated BOOLEAN, \n\tregistered_on DATETIME, \n\trole VARCHAR, \n\tPRIMARY KEY (id), \n\tUNIQUE (email), \n\tCHECK (authenticated IN (0, 1))\n)\n\n']
Traceback (most recent call last):
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1182, in _execute_context
    context)
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 470, in do_execute
    cursor.execute(statement, parameters)
sqlite3.OperationalError: table users already exists

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/bin/flask", line 10, in <module>
    sys.exit(main())
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/flask/cli.py", line 894, in main
    cli.main(args=args, prog_name=name)
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/flask/cli.py", line 557, in main
    return super(FlaskGroup, self).main(*args, **kwargs)
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/flask/cli.py", line 412, in decorator
    return __ctx.invoke(f, *args, **kwargs)
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/flask_migrate/cli.py", line 134, in upgrade
    _upgrade(directory, revision, sql, tag, x_arg)
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/flask_migrate/__init__.py", line 95, in wrapped
    f(*args, **kwargs)
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/flask_migrate/__init__.py", line 280, in upgrade
    command.upgrade(config, revision, sql=sql, tag=tag)
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/alembic/command.py", line 276, in upgrade
    script.run_env()
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/alembic/script/base.py", line 475, in run_env
    util.load_python_file(self.dir, "env.py")
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/alembic/util/pyfiles.py", line 90, in load_python_file
    module = load_module_py(module_id, path)
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/alembic/util/compat.py", line 156, in load_module_py
    spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "migrations/env.py", line 90, in <module>
    run_migrations_online()
  File "migrations/env.py", line 83, in run_migrations_online
    raise exception
  File "migrations/env.py", line 80, in run_migrations_online
    context.run_migrations()
  File "<string>", line 8, in run_migrations
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/alembic/runtime/environment.py", line 839, in run_migrations
    self.get_context().run_migrations(**kw)
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/alembic/runtime/migration.py", line 361, in run_migrations
    step.migration_fn(**kw)
  File "/Users/hjackson/Devel/BEAR-Necessities-Market/migrations/versions/6ca021205583_.py", line 29, in upgrade
    sa.UniqueConstraint('email')
  File "<string>", line 8, in create_table
  File "<string>", line 3, in create_table
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/alembic/operations/ops.py", line 1248, in create_table
    return operations.invoke(op)
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/alembic/operations/base.py", line 345, in invoke
    return fn(self, operation)
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/alembic/operations/toimpl.py", line 101, in create_table
    operations.impl.create_table(table)
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/alembic/ddl/impl.py", line 251, in create_table
    self._exec(schema.CreateTable(table))
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/alembic/ddl/impl.py", line 134, in _exec
    return conn.execute(construct, *multiparams, **params)
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 945, in execute
    return meth(self, multiparams, params)
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/sqlalchemy/sql/ddl.py", line 68, in _execute_on_connection
    return connection._execute_ddl(self, multiparams, params)
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1002, in _execute_ddl
    compiled
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1189, in _execute_context
    context)
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1402, in _handle_dbapi_exception
    exc_info
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 203, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb, cause=cause)
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 186, in reraise
    raise value.with_traceback(tb)
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1182, in _execute_context
    context)
  File "/Users/hjackson/.local/share/virtualenvs/BEAR-Necessities-Market-Z7u6yuCX/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 470, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) table users already exists [SQL: '\nCREATE TABLE users (\n\tid INTEGER NOT NULL, \n\temail VARCHAR, \n\tpassword BLOB, \n\tauthenticated BOOLEAN, \n\tregistered_on DATETIME, \n\trole VARCHAR, \n\tPRIMARY KEY (id), \n\tUNIQUE (email), \n\tCHECK (authenticated IN (0, 1))\n)\n\n']
(BEAR-Necessities-Market) bash-5.0$
```
Is create_all() being called again? If so, `flask db upgrade` will fail because the migrations are trying to create tables that already exist.

FYI, I deleted data.sqlite and tried again with the same result.